Makefile

```diff
@@ -3,7 +3,7 @@ PYTHON_VERSION_VENV ?= python3.8
 .PHONY: venv
 venv:
 	virtualenv --python=${PYTHON_VERSION_VENV} venv
-	venv/bin/pip install --upgrade pip
+	venv/bin/pip install --upgrade 'pip<22'
```

Comment:
why are we pinning pip version?

Reply:
This was just a temp change because pip-compile failed with pip 22. We probably don't have to pin it, I expect a new release of pip-compile will fix that